var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -260,6 +260,13 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
         "KokkosConfigCommon.cmake", relative_root=os.path.join("lib64", "cmake", "Kokkos")
     )
 
+    # sanity check
+    sanity_check_is_file = [
+        join_path("include", "KokkosCore_config.h"),
+        join_path("include", "Kokkos_Core.hpp"),
+    ]
+    sanity_check_is_dir = ["bin", "include", "lib"]
```

Comment:
Kokkos seems to use `GNUInstallDirs`, so `$CMAKE_LIB_DIR` `lib` can be also `lib64` (https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html#result-variables).

Reply:
Ugh, so we are saying we can only support checking for "bin" and "include" in Kokkos Core and only "include" in Kokkos Kernels which is unfortunate but I will make a change to this PR accordingly.